apps/frontend/src/modules/surveys/SurveyProvider.tsx

```diff
@@ -73,20 +86,61 @@ export const SurveyProvider = ({ children }: SurveyProviderProp) => {
     const [deleteSurveyMutation, { loading: pendingDeletion }] =
         useSurveyDeleteMutation();
 
+    const [getSurvey, { loading: loadingSurvey, data: survey }] =
+        useGetSurveyLazyQuery({
+            variables: {
+                slug: surveySlug,
```

Comment:
@kaf-lamed-beyt This variables should be added when you trigger `getSurvey` in the `useEffect`.

Reply:
@jdamilola 

Like the `getSurvey` function should accept args? Or better still, can you show me an example, please?